code/modules/mining/fulton.dm

```diff
@@ -141,14 +137,14 @@ var/list/total_extraction_beacons = list()
 
 /obj/item/fulton_core
 	name = "extraction beacon signaller"
-	desc = "Emits a signal which fulton recovery devices can lock on to. Craft with metal to create a beacon."
+	desc = "Emits a signal which fulton recovery devices can lock on to."
 	icon = 'icons/obj/stock_parts.dmi'
 	icon_state = "subspace_amplifier"
 
 /datum/crafting_recipe/fulton
 	name = "Fulton Recovery Beacon"
 	result = /obj/structure/extraction_point
-	reqs = list(/obj/item/fulton_core = 1, /obj/item/stack/sheet/metal = 5)
```

Comment:
well when you vend it you get 5 metal anyway and most of the time they are just dropped straight in cargo when they are used, so it's basically just an arbitary requirement since you can literally get them at round start